technical-reference.adoc

```diff
@@ -1,512 +1,466 @@
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors. |  |
 |===
- 
+
 === Output
- 
+
 [%autowidth.spread]
 |===
-|Type |Any
-| Attributes Type a| <<HttpResponseAttributes>>
+|Type | Object
 |===
- 
+
+
 === For Configurations
- 
+
 * <<hyperscience-configuration>>
- 
+
 === Throws
- 
-* HYPERSCIENCE:BAD_REQUEST
-* HYPERSCIENCE:CLIENT_ERROR
-* HYPERSCIENCE:CONNECTIVITY
-* HYPERSCIENCE:INTERNAL_SERVER_ERROR
-* HYPERSCIENCE:NOT_FOUND
-* HYPERSCIENCE:SERVER_ERROR
-* HYPERSCIENCE:UNAUTHORIZED
-* HYPERSCIENCE:UNSUPPORTED_MEDIA_TYPE
- 
-[[SubmissionsRetrieve]]
-== Submission Retriete
+
+* `HYPERSCIENCE:BAD_REQUEST`
+* `HYPERSCIENCE:CLIENT_ERROR`
+* `HYPERSCIENCE:CONNECTIVITY`
+* `HYPERSCIENCE:INTERNAL_SERVER_ERROR`
+* `HYPERSCIENCE:NOT_FOUND`
+* `HYPERSCIENCE:SERVER_ERROR`
+* `HYPERSCIENCE:UNAUTHORIZED`
+* `HYPERSCIENCE:UNSUPPORTED_MEDIA_TYPE`
+
+[[submissions-retrieve]]
+== Submission Retrieve
 `<hyperscience:submissions-retrieve>`
- 
- 
-Retrieves a submission using the submission id. This operation makes an HTTP GET request to the api/v5/submissions/{id} endpoint.
- 
+
+Retrieves a submission using the Submission `id`. This operation makes an HTTP GET request to the `api/v5/submissions/{id}` endpoint.
+
 === Parameters
- 
-[%header%autowidth.spread]
+
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
-| id a| String | Submission Id. |  | x
-| flat a| Boolean | Optional parameter that prevents returning the documents, document_folders, unassigned_pages, and rejected_documents arrays of the Submission object. Defaults to true unless specifically passed as false. | False | x
-| Streaming Strategy a| * <<RepeatableInMemoryStream>>
-* <<RepeatableFileStoreStream>>
-* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
-| Custom Query Parameters a| Object | Custom query parameters to include in the request. The specified query parameters are merged with the default query parameters that are specified in the configuration. |  |
-| Custom Headers a| Object | Custom headers to include in the request. The specified custom headers are merged with the default headers that are specified in the configuration. |  |
+| Config Ref a| ConfigurationProvider |  `Name` of the configuration to use to execute this component. |  | x
+| id a| String | Submission `id`. |  | x
+| flat a| Boolean | Optional parameter that prevents returning the `documents`, `document_folders`, `unassigned_pages`, and `rejected_documents` arrays of the Submission object. Defaults to true unless specifically set to `false`. | False | x
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* <<non-repeatable-stream>> |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
```

Comment:
@rui-lomba , what type of repeatable-stream is the default? We have 2.

Reply:
file-store, added!
Thanks